packages/web-pcm-player-react/README.md

```diff
@@ -0,0 +1,47 @@
+# Web PCM player (React)
+
+React bindings for the `@speechmatics/web-pcm-player` package.
+
+# Installation
+
+```
+npm i @speechmatics/web-pcm-player-react
+```
+
+# Usage
+
+Below is an example playing PCM 16 audio rendered as base64. For the full data, see [this example file](../web-pcm-player/example/example.html)
```

Comment:
Could we add a bit of explanation around this for those that won't know what this means? or a link to info on this?

Reply:
An explanation about what specifically?

Reply:
Added some reference links